src/AccessibilityInsights.Desktop/ColorContrastAnalyzer/ColorContrastResult.cs

```diff
@@ -1,139 +1,74 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 using System.Collections.Generic;
-using System.Linq;
 
 namespace AccessibilityInsights.Desktop.ColorContrastAnalyzer
 {
     public class ColorContrastResult
```

Comment:
could you add some comments here on what this class for? 
Name implies what it would be. but it would be great if you add more detail info for others to understand how it works and what it does.